src/App.tsx

```diff
@@ -31,12 +31,10 @@ const StyledApp = styled.div`
 `;
 
 const StyledContent = styled.div`
```

Comment:
```suggestion
const StyledContent = styled(StyledPageWrapperWithMaxWidth)`
```
kunne vi unngått å bruke `StyledPageWrapperWithMaxWidth` overalt slik?

Reply:
nei, da er vi jo tilbake til start igjen der vi har maxWidth... Vi vil ikke ha maxWidth overalt (ref det nye designet)